src/app/activity-history/page.tsx

```diff
@@ -1,7 +1,26 @@
+'use client';
 import React from 'react';
+import CPageCard from 'src/components/CPageCard';
+import ActivityHistoryTable from 'src/containers/ActivityHistoryTable';
```

Comment:
table is not a good name when it's not a table and is a list and also there are more functionality besides the list. So use a more general name for it.